.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "4"
+  - "6.10"
   - "lts/*"
```

Comment:
Maybe you can better put 8 instead.

Reply:
I test `lts` `stable` and minimal version supported.
You think it's not enough ?

Reply:
I mean maybe it's better to name which version you are testing instead of alias to some versions.